server/tests/classes/coolant.test.js

```diff
@@ -0,0 +1,57 @@
+// TODO: Figure out why we need import { Engine } what to do about it
+import { Engine } from '../../src/classes/engine';
+import { System } from "../../src/classes/generic";
+import Coolant from '../../src/classes/coolant';
+
+describe('Coolant', () => {
+
+  test('should throw if called without the \'new\' operator', () => {
+    expect(() => { const c = Coolant(); }).toThrow(/Cannot call a class as a function/);
+  });
+
+  test('should extend System', () => {
+    expect(new Coolant()).toBeInstanceOf(System);
+  })
+
+  describe('constructor', () => {   
+    test('should set default parameters', () => {
+      const cool = new Coolant();
+      expect(cool.name).toBe('Coolant');
+      expect(cool.class).toBe('Coolant');
+      expect(cool.type).toBe('Coolant');
+      expect(cool.coolant).toBe(1);
+      expect(cool.coolantRate).toBe(0.2);
+      expect(cool.transfer).toBeNull();
+    });
+  });
+
+  describe('power', () => {
```

Comment:
The getter and the setter override the `System` class, because the coolant system shouldn't have power requirements. Unit testing this aspect isn't necessary.